source/_includes/playground/header.liquid

```diff
@@ -18,7 +18,16 @@
         <button
           type="button"
           id="playground-copy-url"
-          class="sl-c-button">Copy URL</button>
+          class="sl-c-button sl-c-button--copy sl-c-button--icon-text">
+          <div class="sl-c-playground-button--copy--copy">
+            {%- render 'icons/copy' -%}
+          </div>
+          <div class="sl-c-playground-button--copy--copied">
+            {%- render 'icons/circle-check' -%}
+            <span class="visuallyhidden">URL is Copied</span>
```

Comment:
I kept the message here since the state is only shown visually, this message communicates that the URL has been copied.